earthaccess/api.py

```diff
@@ -161,7 +170,11 @@ def search_services(count: int = -1, **kwargs: Any) -> List[Any]:
     return query.get(hits if count < 1 else min(count, hits))
 
 
-def login(strategy: str = "all", persist: bool = False, system: System = PROD) -> Auth:
+def login(
+    strategy: str = "all",
+    persist: bool = False,
+    system: System = PROD,
+) -> Auth:
```

Comment:
Please add a `Raises` section to the docstring.

Reply:
3e009e6